packages/office-addin-debugging/src/start.ts

```diff
@@ -104,6 +108,11 @@ export async function runDevServer(commandLine: string, port?: number): Promise<
     }
 }
 
+export async function writeProcessIdToFile(id: number): Promise<void> {
```

Comment:
Suggest using the name that reflect more on the purpose of the function rather than its implementation. The purpose is to save the process id for the dev server. While it does this by writing to a file, that doesn't neccessarily need to be in the name. I would recommend having functions `saveDevServerProcessId(id: string)`, `readDevServerProcessId()` and `clearDevServerProcessId()`. That provides functions to save, read, and clear regardless of the implementation (as env var and/or in a file).

You can move the call to set the process id env var into the function, passing in `devServer.pid.toString()`.